crds/roman/specs/combined_specs.json

```diff
@@ -88,6 +88,36 @@
             "tpn":"wfi_flat.tpn",
             "unique_rowkeys":null
         },
+        "mask":{
+            "classes":[
+                "Match",
+                "UseAfter"
+            ],
+            "derived_from":"Hand made 2021-03-25",
+            "extra_keys":null,
+            "file_ext":".asdf",
+            "filekind":"MASK",
+            "filetype":"MASK",
+            "instrument":"WFI",
+            "ld_tpn":"wfi_mask.tpn",
+            "mapping":"REFERENCE",
+            "name":"wfi_mask.rmap",
+            "observatory":"ROMAN",
+            "parkey":[
+                [
+                    "META.INSTRUMENT.DETECTOR"
+                ],
+                [
+                    "META.OBSERVATION.DATE",
+                    "META.OBSERVATION.TIME"
+                ]
+            ],
+            "sha1sum":"8fd9b3bb49c5fa053f11bebf6ac0c10eb32f6e1f",
+            "suffix":"mask",
+            "text_descr":"Mask",
```

Comment:
This is the string that appears on the website in the context summary tables and is usually a little more descriptive.  For JWST this is "Bad Pixel Mask" which I imagine is accurate here as well?